controller.py

```diff
@@ -178,7 +178,14 @@ def run_encoding(
     additional_data: Optional[Any] = None,
 ) -> int:
     session_token = general.get_ctx_token()
-    initial_count = record.count(project_id)
+    initial_count = 0
```

Comment:
could (in the future) be smart to differ between number of vectors created and number of records embedded.

Reply:
this is just for the progress bar.